_config.yml

```diff
@@ -15,12 +15,26 @@ exclude:
 - work
 - ".sass-cache"
 
+# defaults
+defaults:
+  -
+    scope:
+      path: ""
+      type: agencies
+    values:
+      layout: default
+
+collections:
+  agencies:
+    output: true
+    permalink: /:path/
```

Comment:
Yep, I figured using `/:agency` pattern would be easier to remember either way works for me.